### ISSUE TITLE
Add support for aarm64?

### ISSUE THREAD

**Issue by maruthisi** [NVIDIA/libnvidia-container#45]
We use Tegra and Xavier GPUs in Self driving cars. Adding aarm64 support would help us a lot.
Could this be prioritized? 

I saw issue #7 by @Gotrek77 closed in the past. Again requesting.

**Comment by cnrmck** [comment]
We have the same issue, would help our team similarly. 

Specifically, we expected it to exist at the endpoint 
https://nvidia.github.io/libnvidia-container/ubuntu18.04/arm64

**Comment by alexrashed** [comment]
I know it's request number (n+1) for arm64/aarch64. 

But since the other projects ([NVIDIA/nvidia-container-runtime](https://github.com/NVIDIA/nvidia-container-runtime/issues/5) and [NVIDIA/nvidia-docker](https://github.com/NVIDIA/nvidia-docker/issues/214)) are easily built for arm64 with only slight adjustments (4 LOC to be exact) it would just be fair to tell us if you are at least planning to integrate your very own developer boards (for me it's the TX2).

To give you a motivation: I would love to integrate the TX2 into an experimental Kubernetes cluster. I'm quite sure that's a promising use-case for all your new devices.

**Comment by klueska** [comment]
This is the support matrix for the most recent packages (this does not include [Jetson](https://developer.nvidia.com/embedded/develop/hardware) though):
```
+----------------------+-------------+----------------+---------+-----------------+
|  OS Name / Version   |  Identifier | amd64 / x86_64 | ppc64le | arm64 / aarch64 |
+======================+=============+================+=========+=================+
| Amazon Linux 1       | amzn1       |       X        |         |                 |
| Amazon Linux 2       | amzn2       |       X        |         |                 |
| Amazon Linux 2017.09 | amzn2017.09 |       X        |         |                 |
| Amazon Linux 2018.03 | amzn2018.03 |       X        |         |                 |
| Open Suse Leap 15.0  | sles15.0    |       X        |         |                 |
| Open Suse Leap 15.1  | sles15.1    |       X        |         |                 |
| Debian Linux 9       | debian9     |       X        |         |                 |
| Debian Linux 10      | debian10    |       X        |         |                 |
| Centos 7             | centos7     |       X        |    X    |                 |
| Centos 8             | centos8     |       X        |    X    |        X        |
| RHEL 7.4             | rhel7.4     |       X        |    X    |                 |
| RHEL 7.5             | rhel7.5     |       X        |    X    |                 |
| RHEL 7.6             | rhel7.6     |       X        |    X    |                 |
| RHEL 7.7             | rhel7.7     |       X        |    X    |                 |
| RHEL 8.0             | rhel8.0     |       X        |    X    |        X        |
| RHEL 8.1             | rhel8.1     |       X        |    X    |        X        |
| RHEL 8.2             | rhel8.2     |       X        |    X    |        X        |
| Ubuntu 16.04         | ubuntu16.04 |       X        |    X    |                 |
| Ubuntu 18.04         | ubuntu18.04 |       X        |    X    |        X        |
| Ubuntu 19.04         | ubuntu19.04 |       X        |    X    |        X        |
| Ubuntu 19.10         | ubuntu19.10 |       X        |    X    |        X        |
| Ubuntu 20.04         | ubuntu20.04 |       X        |    X    |        X        |
+----------------------+-------------+----------------+---------+-----------------+
```

Please let us know if this resolves your issue.

**Comment by elezar** [comment]
@alexrashed if your intent is to use Jetson devices with the NVIDIA Container Library packages have been released that cover this use case. These should be included in recent Jetpack releases.

Also, as mentioned in https://github.com/NVIDIA/libnvidia-container/issues/135#issuecomment-981704944 the latest (experimental) `1.7.0-rc.1` release of the NVIDIA Container Toolkit adds better packaging support for this in our [standard repositories](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#installing-on-ubuntu-and-debian).

With this in mind and since the last comment is already pretty old, I'm closing this issue. Please reopen or create a new issue if you're still having problems.

**Comment by elezar** [comment]
@alexrashed the `rc.1` packages have been promoted to GA and are available in our `stable` repositories. 

see https://github.com/NVIDIA/nvidia-container-toolkit/releases/tag/v1.7.0

**Comment by Nizarft** [comment]
> @alexrashed the `rc.1` packages have been promoted to GA and are available in our `stable` repositories.
> 
> see https://github.com/NVIDIA/nvidia-container-toolkit/releases/tag/v1.7.0

I'm still having the same issue on 1.7.0

Any hopes it will be solved in 1.8.0-rc1?

**Comment by elezar** [comment]
@Nizarft what issue are you having? If you are unable to install the `arm64` variants of the packages on your devices please provide more information on your setup so that we can address the issues.
